crates/interfaces/src/provider.rs

```diff
@@ -25,7 +25,7 @@ pub enum ProviderError {
     },
     /// The block number was found for the given address, but the changeset was not found.
     #[error("Account {address:?} ChangeSet for block #{block_number} does not exist")]
-    AccountChangesetNotFound {
+    AccountChangeSetsNotFound {
```

Comment:
these errors are about one particular changeset, not multiple changesets
```suggestion
    AccountChangeSetNotFound {
```

Reply:
same for storage changesets